amethyst_audio/src/systems/audio.rs

```diff
@@ -30,24 +30,21 @@ use crate::{
 
 /// Builds an `AudioSystem`.
 #[derive(Default, Debug, new)]
-pub struct AudioSystemDesc {
-    /// Audio `Output`.
-    pub output: Output,
-}
+pub struct AudioSystemDesc;
 
 impl<'a, 'b> SystemDesc<'a, 'b, AudioSystem> for AudioSystemDesc {
     fn build(self, world: &mut World) -> AudioSystem {
         <AudioSystem as System<'_>>::SystemData::setup(world);
 
-        world.insert(self.output.clone());
+        world.insert(Output::default());
```

Comment:
Being able to support a non default output is really important to gamers. This is an unfortunate limitation we should try and work around.

Reply:
nod -- though it was never possible (`Output` was always private in `AudioBundle`), and I think `cpal` or `rodio` will be the layer that exposes different output devices. Would prefer to wait for that instead of handle it at this layer.